Software/Python/easygopigo3.py

```diff
@@ -126,6 +143,87 @@ def __init__(self, use_mutex=False):
         self.right_eye_color = (0, 255, 255)
         self.use_mutex = use_mutex
 
+    def load_robot_constants(self, config_file_path="/home/pi/Dexter/gpg3_config.json"):
+        """
+        Load wheel diameter and wheel base width constants for the GoPiGo3 from file.
+
+        This method gets called by the constructor.
+
+        :param str config_file_path = "/home/pi/Dexter/gpg3_config.json": Path to JSON config file that stores the wheel diameter and wheel base width for the GoPiGo3.
+        :raises FileNotFoundError: When the file is non-existent.
+        :raises KeyError: If one of the keys is not part of the dictionary.
+        :raises ValueError: If the saved values are not positive numbers (floats or ints).
+        :raises TypeError: If the saved values are not numbers.
+        :raises IOError: When the file cannot be accessed.
+        :raises PermissionError: When there are not enough permissions to access the file.
+
+        Here's how the JSON config file must look like before reading it. Obviously, the supported format is JSON so that anyone can come in
+        and edit their own config file if they don't want to go through saving the values by using the API.
+
+        .. code-block:: json
+        
+            {
+                "wheel-diameter": 66.5,
+                "wheel-base-width": 117
+            }
+
+        """
+        with open(config_file_path, 'r') as json_file:
+            data = json.load(json_file)
+            if data['wheel-diameter'] > 0 and data['wheel-base-width'] > 0:
+                self.WHEEL_DIAMETER = data['wheel-diameter']
```

Comment:
Could you simply do a call to set_robot_constants() here?

Reply:
Yes, that would actually be better. DRY principle.